packages/common/src/utils/challenges.ts

```diff
@@ -119,12 +119,12 @@ export const challengeRewardsConfig: Record<
       `Listen to one track a day for seven days to earn ${challenge?.amount} $AUDIO.`,
     fullDescription: () =>
       'Sign in and listen to at least one track every day for 7 days',
-    progressLabel: '%0/%1 Days',
+    progressLabel: '0/1 Days',
```

Comment:
why is this hard coded to 0/1?

Reply:
oh yeah wut..

Reply:
this code is very old too.. confusing.

Reply:
the % should do some interpolation probably, that's why it was there ... like 1st arg 2nd arg thing

Reply:
ohhhh ok